pontoon/teams/static/js/bugzilla.js

```diff
@@ -76,6 +81,7 @@ var Pontoon = (function (my) {
                     '<th class="summary">Summary</th>' +
                     '<th class="last-changed">Last Changed</th>' +
                     '<th class="date-created">Date Created</th>' +
+                    '<th class="person-assigned-to">Assignee</th>' +
```

Comment:
person-assigned-to -> assigned-to
Assignee -> Assigned To

Reply:
okay, thanks.